static/sass/style.scss

```diff
@@ -2161,6 +2161,16 @@ $colors: $blue, $psf, $yellow, $green, $purple, $red;
     background-color: #808080;
 }
 
+div.note {
+    background-color: #eee;
+    border: 1px solid #ccc;
+    padding: 5px 10px;
+}
+
+p.admonition-title {
+    margin: 0px 10px 5px 0px;
```

Comment:
Also, using `px` would break rendering in other devices. Please use the `px2em()` helper.